documentation/components.md

```diff
@@ -11,75 +11,137 @@ const Parent = () => (
   </section>
 );
 
-const Label = ({ greeting, children }) => (
+const Label = (props) => (
   <>
-    <div>{greeting}</div>
-    {children}
+    <div>{props.greeting}</div>
+    {props.children}
   </>
 );
 ```
 
-Since all nodes from JSX are actual DOM nodes the only responsibility of top level Templates/Components is appending to the DOM.
+Since all nodes from JSX are actual DOM nodes, the only responsibility of top level Templates/Components is appending to the DOM.
 
-Components also support dynamic bindings which allow you to pass values that will change. However you need to be careful to access your props inside bindings or effects if you want them to track change.
+## Props
+
+Much like React, Vue, Angular, and other frameworks, you can define properties on your components which allow a parent component to pass data to a child component. Here a parent is passing the string "Hello" to the `Label` component via a `greeting` property.
 
 ```jsx
-// Name will never update as it is destructured outside
-const StaticComponent = ({ name }) => <div>{ name }</div>
+const Parent = () => (
+  <section>
+    <Label greeting="Hello">
+      <div>John</div>
+    </Label>
+  </section>
+);
+```
 
-// Updates like you'd expect
-const DynamicComponent = props => <div>{ props.name }</div>
+In the above example, the value set on `greeting` is static, but we can also set dynamic values. For example:
 
-// Update on state.name change
-<DynamicComponent name={ state.name }/>
+```jsx
+const Parent = () => (
+  let value = Math.random().toString();
 
-// will not update on name change and pass by value
-const { name } = state;
-<DynamicComponent name={ name }/>
+  <section>
+    <Label greeting={value}>
+      <div>John</div>
+    </Label>
+  </section>
+);

```

Comment:
Small syntax mistake here:
```diff
- const Parent = () => (
+ const Parent = () => {
    let value = Math.random().toString();

+   return (
      <section>
        <Label greeting={value}>
          <div>John</div>
        </Label>
      </section>
    );
+ };
```
Also IMO this isn't a great example for dynamic props (in reactive terms), because this prop won't change, but ¯\\\_(ツ)_/¯

Reply:
I completely agree. I'll admit it was a bit of laziness on my part. I figured the point would be made that you can pass variables via props. I figure the docs are an iterative process, and at some point in the future they can be improved further.